src/Shell/Task/MigrationSnapshotTask.php

```diff
@@ -99,8 +99,21 @@ protected function markSnapshotApplied($path)
             $version
         ];
 
+        $dispatchCommand = 'migrations mark_migrated ' . $version;
+        if (!empty($this->params['connection'])) {
+            $_SERVER['argv'][] = '-c';
+            $_SERVER['argv'][] = $this->params['connection'];
+            $dispatchCommand .= ' -c ' . $this->params['connection'];
+        }
+
+        if (!empty($this->params['plugin'])) {
+            $_SERVER['argv'][] = '-p';
+            $_SERVER['argv'][] = $this->params['plugin'];
```

Comment:
Yeah me too.
I looked it up a bit and it is currently the only way I found.
I'll give it another shot just in case.